templates/index.html

```diff
@@ -36,21 +17,22 @@
 
     <div class="meetup">
 
-      <h3 class='meetup-name'><a href="{{ meetup.meetup_link }}">{{ meetup.name }}</a></h3>
+      <h3 class='meetup-name'><a href="{{ meetup.get_meetup_link }}">{{ meetup.get_meetup_name }}</a></h3>
```

Comment:
Try to separate semantic markup from style: use `h3` only if there's already an `h1` and `h2` on the page, to indicate that this is the third-most-important header. Don't use it because it has the right default size. Those presentation concerns are better handled in CSS.